ci/gitlab/.gitlab-ci.yml

```diff
@@ -265,3 +265,11 @@ pages:
   # would be nice to have on a per-PR basis: # https://gitlab.com/gitlab-org/gitlab-ce/issues/35141
   only:
     - master
+
+deploy docker image:
+  stage: deploy
+  script:
+    - ./docker/docker-build.sh
+    - docker push advancedtelematic/aktualizr-app:latest
```

Comment:
As we've talked about it, we've yet haven't added any access to dockerhub from gitlab. If you can work with the gitlab's docker registry that would make things slightly easier.

And it would be good to have a way to try the stage. Maybe with a `only: variables: - $xxx` clause, or pushed without `only: - master` in some other branch.